.travis.yml

```diff
@@ -2,21 +2,32 @@ language: go
 go:
   - "1.11.x"
 sudo: required
-dist: xenial
 
 services:
 - docker
 
 jobs:
   include:
-  - stage: Tests
+  - stage: tests
+    name: Windows Unit Tests
+    os: windows
+    install:
+    - choco install python2
+    script:
+    - enterprise-suite/scripts/lbc_test.py
+ 
+  - stage: tests
     name: Unit tests
+    os: linux
+    dist: xenial
     install:
     - travis_retry scripts/setup-tools-for-ubuntu.sh
     script:
     - make test
-
+ 
```

Comment:
What is the extraneous space I see before me?!
(and again on line 159)

Reply:
Sorry!